modules/nw-udn-additional-config-details.adoc

```diff
@@ -20,12 +20,25 @@ The following table explains additional configurations for UDN that are optional
 
 The `joinSubnets` field configures the routing between different segments within a user-defined network. Dual-stack clusters can set 2 subnets, one for each IP family; otherwise, only 1 subnet is allowed. This field is only allowed for the `Primary` network.
 
-|`spec.IPAMLifecycle`
+|`spec.ipam.lifecycle`
```

Comment:
This is new to me.
Until now the field in our UDN resources was `spec.layer2.ipamLifecycle`, and this setting still produces valid UDNs, which eventually end-up with running VMs with fully functional primary UDN interfaces.
Was that changed? @maiqueb ?
@Anatw @sbahar619 Are you aware of this?

Reply:
Thank you for the clarification @maiqueb .
I have modified my UDN resource according to the doc, and verified the configuration works E2E (including IP and connectivity persistence after migration)